### PR TITLE
chore(cli): implement Validate for `svc override`

### PR DESCRIPTION
**Pull request by efekarakus**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Review by Lou1415926 — APPROVED** [review]
LGTM :shipit:

File: internal/pkg/cli/svc_override.go [hunk 1]
```diff
@@ -73,6 +98,35 @@ func (o *overrideSvcOpts) RecommendActions() error {
...
+		if o.cdkLang == valid {
+			return nil
+		}
+	}
+	return fmt.Errorf("%q is not a valid CDK language: must be one of: %s",
+		o.cdkLang,
+		strings.Join(applyAll(validCDKLangs, strconv.Quote), ", "))
+}
+
+func (o *overrideSvcOpts) appConfig() (*config.Application, error) {
```
Comment (paragbhingre) [inline internal/pkg/cli/svc_override.go]: Are we going to use this method somewhere else if not then just keeping return type as error would be fine I think. We do not need to return `config.Application` if we aren't planning to use it.

**Review by paragbhingre — COMMENTED** [review]
Just one question, otherwise looks good to me.

File: internal/pkg/cli/svc_override.go (hunk 1 above)
Comment (efekarakus) [reply internal/pkg/cli/svc_override.go]: Sure, I removed the `appConfig` method and inlined it instead

**Review by paragbhingre — APPROVED** [review]

**Pull request merged**